src/index.html

```diff
@@ -9,6 +9,9 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
+      <div class="hand-second"></div>
```

Comment:
```suggestion
    <div class="stopwatch">
      <div class="stopwatch__hand-second"></div>
```
![image](https://github.com/mate-academy/layout_stop-watch/assets/96409563/6877c7ed-9c23-4e0e-9942-409e82d891c1)